tests/visualization_tests/matplotlib_tests/test_contour.py

```diff
@@ -105,11 +111,15 @@ def fail_objective(_: Trial) -> float:
         plot_contour(study, ["optuna", "Optuna"])
 
     figure = plot_contour(study, params=params)
+    plt.savefig(BytesIO())
+
     if params is not None and len(params) < 3:
         if len(params) <= 1:
             assert len(figure.get_lines()) == 0
+            plt.savefig(BytesIO())
```

Comment:
Do we need this?
```suggestion
```

Reply:
Sorry for this, I didn't recheck the code.